orchagent/port/porthlpr.cpp

```diff
@@ -740,6 +753,66 @@ bool PortHelper::parsePortDescription(PortConfig &port, const std::string &field
     return true;
 }
 
+bool PortHelper::parsePortPtIntfId(PortConfig &port, const std::string &field, const std::string &value) const
+{
+    SWSS_LOG_ENTER();
+
+    try
+    {
+        if (value != "None")
+        {
+            port.pt_intf_id.value = to_uint<std::uint16_t>(value);
+        }
+        else
+        {
+            /*
+             * In SAI, Path Tracing Interface ID 0 means Path Tracing disabled.
+             * When Path Tracing Interface ID is not set (i.e., value is None),
+             * we set the Interface ID to 0 in ASIC DB in order to disable
+             * Path Tracing on the port.
+             */
+            port.pt_intf_id.value = 0;
+        }
+        port.pt_intf_id.is_set = true;
+    }
+    catch (const std::exception &e)
```

Comment:
should we reject this for port-channel?
Also check for the path-tracing interface id within the valid range. I believe it is between 1-4K as per HLD. Please add this case in swss tests.

Reply:
@kperumalbfn I did the requested changes:
* added a check to reject Path Tracing configuration on port channel
* added a check to ensure Path Tracing interface ID is in the correct range [1, 4095]
* added test cases to verify that OrchAgent behaves as expected when it receives an invalid Path Tracing configuration